cornflow-server/Dockerfile

```diff
@@ -26,19 +26,12 @@ ENV PYTHONUNBUFFERED 1
 
 # install dependencies
 RUN pip install --upgrade pip
-COPY ./requirements.txt /usr/src/app/requirements.txt
-RUN pip install -r requirements.txt --force-reinstall
-
-# copy project
-COPY cornflow /usr/src/app/cornflow
-COPY docs /usr/src/app/docs
-COPY migrations /usr/src/app/migrations
-COPY examples /usr/src/app/examples
-COPY *.py /usr/src/app/
+RUN pip install "git+https://github.com/baobabsoluciones/cornflow@feature/package#subdirectory=cornflow-server"
```

Comment:
This has to be changed before merging and we have to add here a cornflow version code